themes/tiddlywiki/vanilla/base.tid

```diff
@@ -2186,3 +2186,7 @@ body.tc-dirty span.tc-dirty-indicator, body.tc-dirty span.tc-dirty-indicator svg
 	background: #f00;
 	color: #fff;
 }
+
+.tc-keyboard {
+	display: inline;
```

Comment:
> for some reason

was referring to it having to be a block-level element... but other than that, I figured you didn't want to attach the keyboard handling to some parent element